core/src/test/java/org/infinispan/xsite/irac/IracExponentialBackOffTest.java

```diff
@@ -109,7 +109,8 @@ public void testNoBackoffOnOtherException(Method method) throws InterruptedExcep
    }
 
    private void doTest(Method method, Supplier<Throwable> throwableSupplier) throws InterruptedException {
-      backOff.drainPermits();
+      backOff.release();
```

Comment:
Wouldn't these first 4 lines fit better in an `@AfterMethod` method?

Reply:
done